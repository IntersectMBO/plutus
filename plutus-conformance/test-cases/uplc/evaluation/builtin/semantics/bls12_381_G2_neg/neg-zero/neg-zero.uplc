-- The negative of the zero point is the zero point.
(program 1.0.0
 [
  (builtin bls12_381_G2_neg)
  (con bls12_381_G2_element 0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000)
 ]
)
