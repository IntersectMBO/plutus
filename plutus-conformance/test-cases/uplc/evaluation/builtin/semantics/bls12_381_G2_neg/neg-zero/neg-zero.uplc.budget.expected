({cpu: 332646
| mem: 436})